src/index.html

```diff
@@ -4,9 +4,311 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Price:</p>
+          <p class="card__price__value">$2,199</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Buy
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Price:</p>
+          <p class="card__price__value">$2,199</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Buy
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Price:</p>
+          <p class="card__price__value">$2,199</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Buy
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
```

Comment:
**Modifiers**: Modifiers should reflect a state or a variant of a block or element. In your code, _.card__stars--4_ is used to represent a rating. This is acceptable, but consider using a more descriptive name, like _.card__stars--rating-4_.